core/admin/GearAdmin.py

```diff
@@ -15,20 +15,23 @@ class GearAdmin(ViewableModelAdmin):
     list_display = ("name", "status", "get_department", "checked_out_to", "due_date")
 
     # Choose which fields appear on the side as filters
-    list_filter = ('status', "geartype__department", "geartype")
+    list_filter = ("status", "geartype__department", "geartype")
 
     # Choose which fields can be searched for
-    search_fields = ('geartype__name', 'gear_data', 'rfid', 'checked_out_to__first_name', 'checked_out_to__last_name')
+    search_fields = (
+        "geartype__name",
+        "gear_data",
+        "rfid",
+        "checked_out_to__first_name",
+        "checked_out_to__last_name",
+    )
 
     fieldsets = [
-        ('Gear Info', {
-            'classes': ('wide',),
-            'fields': ("rfid", "geartype", "image"),
-        }),
-        ('Checkout Info', {
-            'classes': ('wide',),
-            'fields': ("status", "checked_out_to", "due_date")
-        }),
```

Comment:
From the Black (our new formatter) documentation:
Black prefers double quotes (" and """) over single quotes (' and '''). It will replace the latter with the former as long as it does not result in more backslash escapes than before.